vassal-app/src/main/java/VASSAL/tools/SequenceEncoder.java

```diff
@@ -223,6 +223,15 @@ public Decoder(String value, char delimiter) {
       stop = val != null ? val.length() : 0;
     }
 
+    /**
+     * Decoder constructor that accepts a string for the delimiter (so that a single constant value can be defined for a module, e.g. private static final String DELIMITER = "\t";)
```

Comment:
I'm not grasping what this is for. What prevents defining a `private static final char DELIMITER = '\t';`?

Reply:
So the problem is that some (all?) of the other places the delimiter is used, you actually want/need _String_ not _char_. Meaning that for your trouble of trying to do a good job defining your delimiter rather than just hardcoding it everywhere, you end up getting punished by either having to define two different names (e.g. DELIMITER_STRING and DELIMITER_CHAR) -- which is both bad and inconvenient, or you have to manually convert the String to a char when it is used, which is also kind of bad. So really the encoder should just accept the same String version of the delimiter that is most useful for all other purposes where these are coming up. (see the various uses of DELIMITER in this PR for a tour).